Documentation/contributing/development/images.rst

```diff
@@ -12,30 +12,6 @@ Building Container Images
 Two make targets exists to build container images automatically based on the
 locally checked out branch:
 
-Developer images
-~~~~~~~~~~~~~~~~
-
-Run ``make dev-docker-image`` to build a cilium-agent Docker image that
-contains your local changes.
-
-::
-
-    DOCKER_DEV_ACCOUNT=quay.io/myaccount DOCKER_IMAGE_TAG=jane-developer-my-fix make dev-docker-image
```

Comment:
The linked Slack thread refers to `docker-cilium-image-for-developers` and this pull request is removing documentation for `dev-docker-image`. Those are different targets as far as I can tell.

Reply:
@pchaigno you're right . PR can be closed then ;-)